client/src/App.js

```diff
@@ -15,33 +15,33 @@ import Footer from './components/Footer/Footer';
 import Store from './state-management/Store';
 
 function App() {
-  const { token, setToken } = useToken();
+	const { token, setToken } = useToken();
 
-  if(!token) {
-    return <Login setToken={setToken} />
-  }
+	if (!token) {
```

Comment:
hey Phil. it looks like uncommenting thse few lines prevent us from accessing the other routes. Did it do the same for you?